src/lib/stringIndices.js

```diff
@@ -0,0 +1,16 @@
+  let stringIndices = (userString, userNumber) => {
+    let newArr = []
+    let separate = ""
```

Comment:
same as newArr, remove because redefined below. If these were intended to filter or screen for user input errors and edge cases you need to filter those out using if statements that return null or error messages if the user does not input a string and a number.